### ISSUE TITLE
Can't open WYSIWYG editor

### ISSUE THREAD

**Issue by hulzer** [FlowFuse/node-red-dashboard#1505]
Hi,

I'd like to test the new layout editor feature, but I can't get it working.

When I try to a new page, I can't select the wysiwyg layout from dropdown:
![image](https://github.com/user-attachments/assets/0edc7371-5bc2-4503-9e3a-4dc100487800)

node-red v4.0.2
node-red-dasboard v1.9.1

Thanks.

**Comment by joepavitt** [comment]
As @colinl has mentioned, it is available in the sidebar, but we could add something into that config tray too. Will keep this issue open in order to capture that.

**Comment by bartbutenaers** [comment]
Hi everybody,

I wanted to have a look at this issue, but I don't fully understand how it works.

Suppose a page has layout type xxx (e.g. notebook, ...).  You open the layout in wysiwyg editor, and deploy your changes after chaning the layout.  Then the layout - which arrives in the frontend - still seems to be the original type xxx.

So it looks to me that there is no dedicated *"wysiwyg layout"* type?

Thanks for illuminating me!
Bart

**Comment by joepavitt** [comment]
> So it looks to me that there is no dedicated "wysiwyg layout" type?

That's correct, there was in our original PoC, but byb the time it made it to `main`, @Steve-Mcl had done some great work so that actually, all of the existing layouts are editable through a WYSIWYG editor.

So, a button alongside the layout type editor of "Edit", could then open that WYSIWYG editor for the page in the browser? I had also considered an "Edit Layout" button at the top of the tray, alongside where the existing "Delete" button is, but that could be available for _every_ node - making it easier to jump into layout editing from any point.
